### ISSUE TITLE
.param is still exist after 1st decompile even when use -b command

### ISSUE THREAD

**Issue by Arsyadi** [JesusFreke/smali#317]
I already reported in apktool xda thread, but i think its from smali itself.
Look here http://forum.xda-developers.com/showpost.php?p=62786765&postcount=3627
But its removed on 2nd decompile (1st decompile, then compile, then use -b command again in 2nd decompile).

**Comment by JesusFreke** [comment]
Issue description copied from the xda-dev post


Originally Posted by @Arsyadi

----

Yes, i already used -b command, but still get .param strings (only .param) like this

Code:
```
# virtual methods
.method protected executor(Lcom/google/common/util/concurrent/Service$State;)Ljava/util/concurrent/Executor;
    .locals 1
    .param p1    # Lcom/google/common/util/concurrent/Service$State;

    new-instance v0, Lcom/google/common/util/concurrent/AbstractIdleService$2;

    invoke-direct {v0, p0, p1}, Lcom/google/common/util/concurrent/AbstractIdleService$2;-><init>(Lcom/google/common/util/concurrent/AbstractIdleService;Lcom/google/common/util/concurrent/Service$State;)V

    return-object v0
.end method
```
Please fix if its possible because i dont want decompile twice. 



Yes i hope so. SO we can do the job easily.

**Comment by JesusFreke** [comment]
I suspect you may be using an old version of smali/baksmali. Can you try with v2.1.0?

**Comment by JesusFreke** [comment]
Nevermind, I see the case in the current code when this can happen.

**Comment by Arsyadi** [comment]
I already use latest 2.1.0 , but still same. 1st decompile can't remove it, but 2nd decompile remove it, its strange.

**Comment by JesusFreke** [comment]
This should be fixed by d6b17b99. I'll likely be releasing a new v2.1.1 soon, which will contain this fix.
